package.json

```diff
@@ -7,7 +7,7 @@
   "main": "src/index.js",
   "scripts": {
     "start": "cross-env NODE_ENV=development electron .",
-    "lint": "ts-standard && standard",
+    "lint": "aegir lint",
```

Comment:
💭 the fact we are adding ~30k of lines to the `package-lock.json` gives me a pause. it is pulling a lot of dev dependencies.

the only reason aegir is pulled is linter - it does not sound right. 
can the linter be updated without pulling in aergir?

Reply:
Thanks for providing feedback. I do really appreciate it. 

But I was advised to add/use Aegir. I think that the middle ground here is to remove aegir after we agree on the other commits.